src/app.py

```diff
@@ -63,21 +64,22 @@ def start_detection():
         TOKEN = os.getenv("HARD_TOKEN")
         RESOURCE_PATH = os.getenv("RESOURCE_PATH")
         URL = f"{domain}/api/v2/tickets"
+
         for image_path in test_data_paths:
-            sleep(5)
+            sleep(15)
             detection_result = detector.detect(f"{RESOURCE_PATH}/{image_path}")
             detection_result = os.linesep.join([s for s in detection_result.splitlines() if s and len(s) > 1])
             detection_result = detection_result.replace(" ", "")
             date_iso = datetime.now().isoformat()
             data = {
-                "parkingLotId": 1,
+                "parkingLotId": 7,
                 "plate": detection_result,
-                "createdAt": date_iso,
-                "isItInside": True,
+                "createdAt": date_iso
```

Comment:
There is 2 `createdAt` variable in `data` body.

Reply:
Red line means deleted. Green line means added.

Reply:
Sure, why not ashdhsd